### ISSUE TITLE
What is the difference between 'pair' and 'simple_pair'

### ISSUE THREAD

**Issue by ShiguangSun** [zchuning/repo#1]
Hi, I'm interested in this work. I find that 'calibration_mode' has two forms, 'pair' and 'simple_pair', and the default is 'simple_pair'.  What's the difference between them?

And I find that calibration loss in code seems to be a cross-entropy loss, rather than the L2 loss mentioned in the paper. Why do this? Will it improve performance?

**Comment by zchuning** [comment]
The difference between "pair" and "simple_pair" is in the calibration. "pair" matches the latent states encoded by the latent dynamics model, whereas "simple_pair" matches the raw visual features outputted by the visual encoder. We found "simple_pair" to work slightly better than "pair," and we used "simple_pair" for the experiments in the paper.

Notice in line 446 of `repo_adapt.py`, we construct a gaussian distribution with unit variance. Maximizing the log probability under this gaussian distribution is equivalent to minimizing the L2 distance.
